2025-03-15 15:23:45,944 INFO    MainThread:180016 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2025-03-15 15:23:45,967 INFO    MainThread:180016 [wandb_setup.py:_flush():76] Configure stats pid to 180016
2025-03-15 15:23:45,968 INFO    MainThread:180016 [wandb_setup.py:_flush():76] Loading settings from /home/dennis00/.config/wandb/settings
2025-03-15 15:23:45,969 INFO    MainThread:180016 [wandb_setup.py:_flush():76] Loading settings from /home/dennis00/cortexGPT/wandb/settings
2025-03-15 15:23:45,970 INFO    MainThread:180016 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2025-03-15 15:23:45,971 INFO    MainThread:180016 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-03-15 15:23:45,972 INFO    MainThread:180016 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-03-15 15:23:45,974 INFO    MainThread:180016 [wandb_init.py:_log_setup():526] Logging user logs to /home/dennis00/cortexGPT/wandb/run-20250315_152345-69qr4b04/logs/debug.log
2025-03-15 15:23:45,975 INFO    MainThread:180016 [wandb_init.py:_log_setup():527] Logging internal logs to /home/dennis00/cortexGPT/wandb/run-20250315_152345-69qr4b04/logs/debug-internal.log
2025-03-15 15:23:45,976 INFO    MainThread:180016 [wandb_init.py:init():566] calling init triggers
2025-03-15 15:23:45,976 INFO    MainThread:180016 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'seed': 0, 'dataset_name': 'ms', 'do_train': True, 'load_model': '../scGPT_data/scGPT_human', 'mask_ratio': 0.0, 'epochs': 10, 'n_bins': 51, 'MVC': False, 'ecs_thres': 0.0, 'dab_weight': 0.0, 'lr': 0.0001, 'batch_size': 8, 'layer_size': 128, 'nlayers': 4, 'nhead': 4, 'dropout': 0.2, 'schedule_ratio': 0.9, 'save_eval_interval': 5, 'fast_transformer': True, 'pre_norm': False, 'amp': True, 'include_zero_gene': False, 'freeze': False, 'DSBN': False}
2025-03-15 15:23:45,977 INFO    MainThread:180016 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 8mc7924l
2025-03-15 15:23:45,981 INFO    MainThread:180016 [wandb_run.py:_finish():1970] finishing run dennis0628/scGPT/8mc7924l
2025-03-15 15:23:45,982 INFO    MainThread:180016 [jupyter.py:save_history():445] not saving jupyter history
2025-03-15 15:23:45,983 INFO    MainThread:180016 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2025-03-15 15:23:45,984 INFO    MainThread:180016 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2025-03-15 15:23:45,985 INFO    MainThread:180016 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2025-03-15 15:23:45,986 INFO    MainThread:180016 [wandb_run.py:_restore():2193] restore
2025-03-15 15:23:45,988 INFO    MainThread:180016 [wandb_run.py:_restore():2199] restore done
2025-03-15 15:23:51,956 INFO    MainThread:180016 [wandb_run.py:_footer_history_summary_info():3851] rendering history
2025-03-15 15:23:51,957 INFO    MainThread:180016 [wandb_run.py:_footer_history_summary_info():3883] rendering summary
2025-03-15 15:23:51,972 INFO    MainThread:180016 [wandb_run.py:_footer_sync_info():3810] logging synced files
2025-03-15 15:23:52,106 INFO    MainThread:180016 [wandb_init.py:init():616] starting backend
2025-03-15 15:23:52,106 INFO    MainThread:180016 [wandb_init.py:init():620] setting up manager
2025-03-15 15:23:52,110 INFO    MainThread:180016 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: fork
2025-03-15 15:23:52,111 INFO    MainThread:180016 [wandb_init.py:init():628] backend started and connected
2025-03-15 15:23:52,125 INFO    MainThread:180016 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2025-03-15 15:23:52,126 INFO    MainThread:180016 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-03-15 15:23:52,126 INFO    MainThread:180016 [wandb_init.py:init():720] updated telemetry
2025-03-15 15:23:52,238 INFO    MainThread:180016 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2025-03-15 15:23:53,241 INFO    MainThread:180016 [wandb_run.py:_on_init():2262] communicating current version
2025-03-15 15:23:53,284 INFO    MainThread:180016 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.19.8 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-03-15 15:23:53,285 INFO    MainThread:180016 [wandb_init.py:init():804] starting run threads in backend
2025-03-15 15:24:13,360 INFO    MainThread:180016 [wandb_run.py:_console_start():2241] atexit reg
2025-03-15 15:24:13,361 INFO    MainThread:180016 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2025-03-15 15:24:13,362 INFO    MainThread:180016 [wandb_run.py:_redirect():2161] Wrapping output streams.
2025-03-15 15:24:13,363 INFO    MainThread:180016 [wandb_run.py:_redirect():2186] Redirects installed.
2025-03-15 15:24:13,364 INFO    MainThread:180016 [wandb_init.py:init():847] run started, returning control to user process
2025-03-15 15:27:54,414 INFO    MainThread:180016 [wandb_watch.py:watch():51] Watching
2025-03-15 15:28:29,235 INFO    MainThread:180016 [wandb_watch.py:watch():51] Watching
2025-03-15 15:59:48,047 INFO    MainThread:180016 [wandb_watch.py:watch():51] Watching
